### PR TITLE
Added alert for ceph pg stuck

### PR DESCRIPTION
**Pull request by anmolsachan**
Signed-off-by: Anmol Sachan <anmol13694@gmail.com>

ceph-mixins-bug-id: #8

### CODE REVIEW

File: alerts/pg.libsonnet [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            alert: 'OCS_CephPgStuck',
+            expr: |||
+              max(ceph_osd_numpg{%(cephExporterSelector)s}) - scalar(ceph_pg_active{%(cephExporterSelector)s}) <= 0
+            ||| % $._config,
+            'for': $._config.pgStuckAlertTime,
+            labels: {
+              severity: 'error',
+            },
+            annotations: {
+              message: 'OCS Ceph PG\'s in stuck state. Recommended for Admin to review the cluster.',
```
Comment (shtripat) [inline alerts/pg.libsonnet]: I feel message should be something like `Data reliability issues with storage cluster. Recommended for admin to review the cluster`. @pcuzner comments?

File: alerts/pg.libsonnet [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'pg-alert.rules',
+        rules: [
+          {
+            alert: 'OCS_CephPgStuck',
```
Comment (shtripat) [inline alerts/pg.libsonnet]: Remove the `OCS_` prefix as discussed.

**Comment by shtripat** [comment]
Rebase the PR with latest master

File: alerts/pg.libsonnet [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            expr: |||
+              max(ceph_osd_numpg{%(cephExporterSelector)s}) - scalar(ceph_pg_active{%(cephExporterSelector)s}) <= 0
```
Comment (shtripat) [inline alerts/pg.libsonnet:10]: Shouldn't this rule be like `max(ceph_osd_numpg{job="rook-ceph-mgr"}) - scalar(ceph_pg_active{job="rook-ceph-mgr"}) >= 0` or we need to take rate of this?

@pcuzner comments?

**Comment by shtripat** [comment]
As per latest comments from @pcuzner this PR is not required.
@pcuzner please confirm and we will close this.

**Comment by anmolsachan** [comment]
No required anymore.

**Pull request closed without merging**